.lintstagedrc.mjs

```diff
@@ -0,0 +1,4 @@
+export default {
+  "*.{ts,tsx,mjs,js,jsx,json,html,vue,scss,css,md,yml}": "prettier --write",
+  "**/*.ts": "eslint --fix",
```

Comment:
Shouldn't there be a `git add` at the end?

Reply:
> Shouldn't there be a `git add` at the end?

Not with recent version of `lint-staged`:
> Prior to version 10, tasks had to manually include git add as the final step. This behavior has been integrated into lint-staged itself in order to prevent race conditions with multiple tasks editing the same files. If lint-staged detects git add in task configurations, it will show a warning in the console. Please remove git add from your configuration after upgrading.

https://github.com/okonet/lint-staged#reformatting-the-code